docs/trace/extending-the-sdk/README.md

```diff
@@ -206,6 +206,30 @@ class MyProcessor : BaseProcessor<Activity>
 
 A demo processor is shown [here](./MyProcessor.cs).
 
+### Filtering Processor
```

Comment:
we could ship an official FilterProcessor as part of the SDK, and have exporters provide additional extention methods like builder.UseExporter(ExporterOptions), and add ExporterOptions.Filter. If filter is defined, then wrap the whole processor inside the FilteringProcessor. This allows exporter ctors() to remain internal, and also avoids the provider.dispose issues I mentioned in the PR description

Reply:
I like that idea!

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/1578

Reply:
+1